src/plugins/unified_doc_viewer/public/components/doc_viewer_logs_overview/sub_components/highlight_field_description.tsx

```diff
@@ -17,12 +18,8 @@ export function HighlightFieldDescription({ fieldName }: { fieldName: string })
 
   const title = (
     <EuiFlexGroup alignItems="center" gutterSize="s">
-      {type && (
-        <EuiFlexItem grow={false}>
-          <FieldIcon type={type} size="s" />
-        </EuiFlexItem>
-      )}
-      <EuiFlexItem grow={false}>{fieldName}</EuiFlexItem>
+      {type && <FieldIcon type={type} size="s" />}
+      {fieldName}
```

Comment:
@weltenwort mentioned this already, but I also wouldn't expect to see `EuiFlexGroup` used without `EuiFlexItem`. Was there a reason these changes were made?

Reply:
I explained more in detail on [this comment](https://github.com/elastic/kibana/pull/180262#issuecomment-2074183685).